packages/arb-token-bridge-ui/src/components/common/HeaderAccountPopover.tsx

```diff
@@ -187,128 +152,116 @@ export function HeaderAccountPopover() {
     window.location.reload()
   }
 
+  function openTransactionHistory() {
+    dispatch({ type: 'layout.set_txhistory_panel_visible', payload: true })
+  }
+
+  const headerItemsClassName =
+    'arb-hover flex w-full flex-row items-center space-x-2 px-[4rem] py-2 text-lg lg:text-base font-light text-white hover:bg-blue-arbitrum lg:px-4'
+
+  const setTxHistoryViewed = () => {
```

Comment:
I think the idea is to reuse it in multiple places without having to explicitly set state. Although since it's just one liner with a single purpose I don't think I'd use a wrapper here